google-cloud-spanner/lib/google/cloud/spanner/errors.rb

```diff
@@ -20,6 +20,16 @@ module Cloud
     module Spanner
       ##
       # @private
+      # # CommitError
+      #
+      # Signal that the Transaction has been committed.
```

Comment:
I am not sure adding this extra control flow mechanism is a good idea. Is this common practice in Ruby. I would rather people used the standard mechanisms. Is there not some standard mechanism to do this i.e break out of a loop early.